pkg/resources/common/formatter.go

```diff
@@ -122,6 +123,15 @@ func formatter(summarycache *summarycache.SummaryCache, asl accesscontrol.Access
 		} else {
 			delete(resource.Links, "remove")
 		}
+		if hasPatch {
+			if attributes.DisallowMethods(resource.Schema)[http.MethodPatch] {
+				resource.Links["patch"] = "blocked"
+			} else {
```

Comment:
Please delete the block
```
} else {
  resource.Links["patch"] = u
  ```
  
  This code caused the slack-only issue fixed by https://github.com/rancher/steve/pull/451

Reply:
Thanks.  I've made a change to apiserver to return the patch link.  This PR is using my forked copy of that for now, but it seems to work correctly.